model/project_ref.go

```diff
@@ -242,6 +242,10 @@ var adminPermissions = gimlet.Permissions{
 	evergreen.PermissionLogs:            evergreen.LogsView.Value,
 }
 
+var viewSettingsPermissions = gimlet.Permissions{
```

Comment:
does this need to be a constant? You only use it once and it's a single value so I would just embed it in the branch admin role

Reply:
This comment actually reminded me I didn't explicitly test the route that uses view settings; can you re-review? 😅